TaskModules/powershell/TaskModuleSqlUtility/SqlQueryOnTargetMachines.ps1

```diff
@@ -48,25 +48,65 @@ function Execute-SqlQueryDeployment
         # Import SQLPS Module
         Import-SqlPs
 
-        $scriptArgument = "Invoke-Sqlcmd -ServerInstance `"$serverName`" -Database `"$databaseName`" -InputFile `"$sqlFile`" "
-        $scriptArgument += $additionalArguments
+        $spaltArguments = @{
+            ServerInstance=$serverName
+            Database=$databaseName
+            InputFile=$sqlFile
+        }
+
+        # Process Additional arguments 
+        if($additionalArguments) 
+        {
+            $args = $additionalArguments.Split()
+            $argScope = $null
+            foreach($arg in $args) {
+                $arg = $arg.Trim()
+                if($arg)
+                {
```

Comment:
I believe there is some regular expression in dtl sdk layer which is used to split the string command to dictionary of key value pairs. You can refer DTL SDK code for this and use the same logic which is already tested. 

You can refer code path of ScriptContent with arguments, you can take Chaitanya help to figure out where to start.